clients/crystal/github.com/vladfaust/mini_redis.json

```diff
@@ -1,7 +1,6 @@
 {
     "name": "mini_redis",
     "description": "A light-weight low-level Redis client for Crystal",
-    "homepage": "http://github.vladfaust.com/mini_redis/",
```

Comment:
Shouldn't we just remove this project entirely? If the URL is invalid, the project is probably dead.